### ISSUE TITLE
double negative in tweak mode after saving changes

### ISSUE THREAD

**Issue by james-mcelwain** [processing/processing#4013]
In the case of a variable minus a positive integer, e.g.` x - 5`, when the integer is tweaked to be negative, save changes will result in like `x--5` which throws an error. It would be great if saving changes from tweak mode could wrap the tweaked value in a paren (or convert the double `-` to a `+` ).

Thanks and much love!

**Comment by galsasson** [comment]
Thanks for catching that bug.
Just to make sure, in my environment ```x - -5``` doesn't throw an error but this issue does happen when the two are next to each other like so: ```x--5``` (when something like ```x-1``` becomes ```x--5``` after tweak)
Is that the case in your environment too or do you get an error with the first one as well?

**Comment by james-mcelwain** [comment]
Yup, bad example on my part--should have copied the offending code directly from my sketch: properly spacing the infix operator solves the problem, it's only when there is no whitespace between the `-`s. that an error is thrown.

**Comment by ykabusalah** [comment]
Can I tackle this issue?

**Comment by jeremydouglass** [comment]
Glad you are excited about responding to a `help wanted`!

Have you confirmed that this error is still present in the latest release? Do you have a test case showing the error?

see also: https://github.com/processing/processing/issues/5805
